tests/test_util.py

```diff
@@ -271,6 +271,12 @@ def test_biom_open_hdf5(self):
 
         remove(get_data_path('test_writing.biom'))
 
+    @npt.dec.skipif(HAVE_H5PY is True, msg='Can only be tested without H5PY')
```

Comment:
Fixed, thanks for catching that!

On (Jan-26-15|12:22), Jorge Cañardo Alastuey wrote:
> > @@ -271,6 +271,12 @@ def test_biom_open_hdf5(self):
> >  
> >          remove(get_data_path('test_writing.biom'))
> >  
> > +    @npt.dec.skipif(HAVE_H5PY is True, msg='Can only be tested without H5PY')
> 
> `@npt.dec.skipif(HAVE_H5PY, ...`
> 
> `HAVE_H5PY` is already a boolean :)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/biom-format/pull/586/files#r23561105